scene/animation/animation_mixer.cpp

```diff
@@ -696,14 +696,19 @@ bool AnimationMixer::_update_caches() {
 						track_value->init_value = anim->track_get_key_value(i, 0);
 						track_value->init_value.zero();
 
+						track_value->init_use_continuous = callback_mode_discrete == ANIMATION_CALLBACK_MODE_DISCRETE_FORCE_CONTINUOUS;
```

Comment:
Why does the value of callback_mode_discrete at initialization time affect its blending later?

Line 1428 will use this to calculate `force_continuous` and force `use_continuous = true` at that time.

Reply:
As shown in the table in the description, this is because the process needs to be changed depending on the presence or absence of the reset track.